services/graphite/service.go

```diff
@@ -252,14 +319,23 @@ func (s *Service) processBatches(batcher *tsdb.PointBatcher) {
 	for {
 		select {
 		case batch := <-batcher.Out():
+			if s.writeSkip {
+				s.ep.Add(EP_BATCHES_SKIPPED, 1)
+				continue
+			}
 			if err := s.PointsWriter.WritePoints(&cluster.WritePointsRequest{
 				Database:         s.database,
 				RetentionPolicy:  "",
 				ConsistencyLevel: s.consistencyLevel,
 				Points:           batch,
-			}); err != nil {
+			}); err == nil {
+				s.ep.Add(EP_BATCHES_TRANSMITTED, 1)
```

Comment:
Shouldn't this get incremented even if they fail?  That batches/points were still transmitted.